### DISCUSSION TITLE
Switch on/off Fans (possibly with speed control?

### DISCUSSION THREAD

**Discussion by sean-saunders** [greghesp/ha-bambulab#52] [Ideas]
It would be great if we could control the various fans (Aux, Chamber and Part) from within home assistant as we can in the handy app.

I am very appreciative of being able to see what we can so far, and am happy to test any pre-releases for this feature!

Kind regards,
Sean

**Comment by LayerCakeMakes** [comment]
greghesp has already added this feature to the open issues list #42

**Reply by sean-saunders** [reply]
I saw it was raised for speed control, but not for just on/off status, which hopefully is an easier integration.
